src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,28 +132,34 @@ public boolean hasRentalInformation(Client client, RentalInformation rentalInfor
         return addressBook.hasRentalInformation(client, rentalInformation);
     }
 
-    //=========== Filtered Client List Accessors =============================================================
+    //=========== Filtered and Sorted Client List Accessors ==========================================================
 
     /**
      * Returns an unmodifiable view of the list of {@code Client} backed by the internal list of
      * {@code versionedAddressBook}
      */
     @Override
     public ObservableList<Client> getFilteredPersonList() {
-        return filteredClients;
+        return sortedFilteredClients;
```

Comment:
You may want to consider to use a general name for the list, so it will not be weird that you call getFilteredPersonList() but then sortedFilteredClients is return. Stick with a name then use that for all the related methods e.g. getter and updating the list.